dnf-plugins-core.spec

```diff
@@ -9,7 +10,8 @@
 %endif
 
 Name:           dnf-plugins-core
-Version:        1.0.0
+Epoch:          1
```

Comment:
because it is part of merging some other plugins or such (at least with tracer, not sure about versionlock). Anyway, we can drop those epochs from such provides